stl/inc/hash_map

```diff
@@ -58,6 +58,11 @@ namespace stdext {
         using _Deduce_key = const _Kty&;
         using key_equal   = _Tr;
 
+#if _HAS_CXX20 && defined(__EDG__) // TRANSITION, DevCom-10678753
+        template <class, class>
+        static constexpr bool _Supports_transparency = false;
+#endif // _HAS_CXX20 && defined(__EDG__)

```

Comment:
Let's follow the newer convention of commenting these with `// ^^^ workaround ^^^` as a reminder that this code should be removed.